client-index/src/synchronizer.rs

```diff
@@ -34,7 +37,12 @@ where
     }
 
     /// Synchronizes transaction index for given view key with Crypto.com Chain (from last known height)
-    pub fn sync(&self, view_key: &PublicKey, private_key: &PrivateKey) -> Result<()> {
+    pub fn sync(
```

Comment:
I'm wondering, as these parts are now being rewritten -- could the manual sync make use of batch requests? (perhaps in a different PR)
From TM 0.35, one could make use of https://www.jsonrpc.org/specification#batch https://godoc.org/github.com/tendermint/tendermint/rpc/client
so instead of one JSON-RPC call at a time, one could submit several in an array in one HTTP request

Reply:
yes
using batch will increase sync speed dramatically

also other approaches which eth is using 
make headers download first,
multiple downloading simulatneously.
this is too complicated, we can try in next phase.

Reply:
also sync is the most challenging part in blockchain networking.

Reply:
This can and should be done. Will do this in future PRs.